playbook/app/pb_kits/playbook/pb_table/_table.tsx

```diff
@@ -85,33 +88,109 @@ const Table = (props: TableProps): React.ReactElement => {
         className
     )
 
+    useEffect(() => {
+        const handleStickyColumns = () => {
```

Comment:
this is awesome work @nickamantia ! Only minor code structure feedback I would have is that we can maybe move some of this function to a separate helper file so the main file is less lengthy. We can keep that in mind for the follow up stories, not a blocker here!

Reply:
@nidaqg great idea Nida! :)